server/views/partials/personalPage/languages.njk

```diff
@@ -0,0 +1,42 @@
+{% from "../../macros/summaryListRow.njk" import summaryListRow %}
+{% macro otherLanguagesText(language) %}
+  {% if language.canSpeak and language.canWrite %}
```

Comment:
Do we need to have a `and not <the_other_canSomething>` for the first 3 conditions? i.e. what if all 3 are `true`?

Reply:
Ah yeah you're right - I'll add that in in the next PR